.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+## Overview
+<!--  
+Provide a high-level description of this change.   
+-->
+
+
+
+## Type of change
+<!--  
+Check the box below that describes your change best:
+--> 
+
+- [ ] new plugin
+- [ ] improvement for an existing plugin
+- [ ] bug-fix for an existing plugin
+- [ ] changes in contributor's experience
```

Comment:
Would be nicer to leverage GitHub's built-in PR "type selector", just like we already do now for issues. Can be done simply by adding more YAML entries.

Reply:
My bad, they are still `.md` files, `.yml` is issues only. But you can have multiple types by creating more entries in the `PULL_REQUEST_TEMPLATE` dir, e.g. `.github/PULL_REQUEST_TEMPLATE/new-plugin.md`.

Reply:
I saw a mention of that [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository), however it looks like PR authors would have to add query paramters to their URL manually to load those templates? Looks like there is no native picker or anything like that, which is definitely bad UX, reason I went for centralising these. 😄 
Let me know if there's anything I'm missing!

Reply:
Ah, I think you're right. In that case, ignore this comment.